functions/Get-DbaLogin.ps1

```diff
@@ -20,7 +20,7 @@ function Get-DbaLogin {
         The login(s) to process - this list is auto-populated from the server. If unspecified, all logins will be processed.
 
     .PARAMETER ExcludeLogin
-        The login(s) to exclude - this list is auto-populated from the server
+        The login(s) to exclude from an auto-populated list of logins from the server.
```

Comment:
```suggestion
        The login(s) to exclude. Options for this list are auto-populated from the server.
```

This matches the text we use in other commands for the same parameter (sync and copy commands).